### PR TITLE
Fixed fuzzy tests errors, and made execution faster

### PR DESCRIPTION
**Pull request by tomaszrybicki**

### CODE REVIEW

File: tests/security/fuzzy/test_fuzzy.py [hunk 1]
```diff
@@ -290,8 +258,9 @@ def test_fuzz_procfs():
         TestRun.executor.run_expect_success('killall afl-fuzz && screen -X -S master kill')
 
     # Stop tracing
-    TestRun.executor.run_expect_success(f'kill {fio_pid}')
+    TestRun.executor.run(f'kill {fio_pid}')
```
Comment (Ostrokrzew) [inline tests/security/fuzzy/test_fuzzy.py]: If You want, there's a `kill_process` method for `TestRun.executor`.

Comment (tomaszrybicki) [reply tests/security/fuzzy/test_fuzzy.py]: You are right, however I need to expect success for the second 'kill' (kill_process does not check that). And I prefer to leave the first one as is to make both calls consistent.

File: source/userspace/InterfaceKernelTraceCreatingImpl.cpp [hunk 2]
```diff
@@ -4,17 +4,17 @@
...
-#include <cstdio>
-#include <string>
-#include <thread>
 #include <octf/interface/TraceManager.h>
 #include <octf/plugin/NodePlugin.h>
 #include <octf/proto/trace.pb.h>
 #include <octf/trace/iotrace_event.h>
 #include <octf/utils/Exception.h>
 #include <octf/utils/Log.h>
+#include <sys/types.h>
```
Comment (mariuszbarczak) [inline source/userspace/InterfaceKernelTraceCreatingImpl.cpp:13]: Please use clang formatter according to our coding style.

File: source/userspace/InterfaceKernelTraceCreatingImpl.cpp [hunk 3]
```diff
@@ -84,6 +84,8 @@ void InterfaceKernelTraceCreatingImpl::StartTracing(
         }
     } catch (Exception &e) {
         controller->SetFailed(e.what());
+    } catch (std::exception &e) {
```
Comment (mariuszbarczak) [inline source/userspace/InterfaceKernelTraceCreatingImpl.cpp:87]: In my opinion it should be a pert of another commit.
